dask/tests/test_distributed.py

```diff
@@ -155,7 +155,8 @@ def test_zarr_in_memory_distributed_err(loop):
     da = pytest.importorskip('dask.array')
     zarr = pytest.importorskip('zarr')
     with cluster() as (s, [a, b]):
-        with Client(s['address'], loop=loop) as c:
+        with Client(s['address'], loop=loop,
+                    client_kwargs={'set_as_default': True}) as c:
```

Comment:
Adding this within the test for now. We may opt to drop this if we decide to add this setting for all Clients created by the testing framework. ( https://github.com/dask/distributed/pull/2057 )

Reply:
Hmm...guess this wasn't needed for CI. Maybe something is messed up in my dev environment. 🤔